package.json

```diff
@@ -19,7 +19,7 @@
     "lint:typescript": "eslint -c types/.eslintrc.json types/**/*.d.ts test/types/**/*.test-d.ts",
     "prepublishOnly": "PREPUBLISH=true tap --no-check-coverage test/build/**.test.js",
     "test": "npm run lint && npm run unit && npm run test:typescript",
-    "test:ci": "npm run unit -- -R terse --cov --coverage-report=lcovonly && npm run test:typescript",
+    "test:ci": "npm run unit -- --cov --coverage-report=lcovonly && npm run test:typescript",
```

Comment:
Why are we removing `-R terse`?

Reply:
I guess we can add it back. I couldn't understand what the problem was without the full tap output.

Reply:
I'd be in favour of more output in CI so folks don't need to add this to debug

Reply:
Except 99% of the time we only care about what test has failed and finding it in among the thousands of lines of "good" output is basically impossible.

Reply:
I get that, though, why do you only care about what test has failed? Just knowing what test failed is sometimes enough information to move forward, but you also often have to know how the test failed or the more details. If CI just tells you which test failed, then you need to rerun locally, wasting time because CI already had the information, it just didn't show it to you. I don't feel super strongly, but I feel like CMD+Fing for the failure and having all the information at hand is kinda nice, instead of making humans toil more to reproduce that information. 

Something we've done at my company is log the verbose output to a file and have that uploaded as a test artifact as well.

Reply:
Try out `-R terse` with a failing test.